utils/git.ts

```diff
@@ -15,11 +15,12 @@ export async function clone(url: string, tag: string): Promise<string> {
       url,
       tmp,
     ],
-    stdout: "piped",
-    stderr: "piped",
+    stdout: "null",
+    stderr: "null",
```

Comment:
Seems like it could cause some unfortunate silent failures. What about just inheriting the stdio from the parent?